docs/source/guides/user/chapters/logging.rst

```diff
@@ -19,6 +19,7 @@ streams) are listed below:
 
 :app: The text based UI (avocado.app)
 :test: Output of the executed tests (avocado.test, "")
+:job: Avocado job related logs (avocado.job)
```

Comment:
Do you think we can also add an official namespace for external libraries used by the tests here? Something like `avocado.extlib`? Or could we e.g. add a custom stream in some easy way?

Reply:
I don't think that it is necessary. You can add external libraries logs by adding them to `avocado` namespace. For example, if you want to have them as part of job.log you will add them to `avocado.job` namespace if you want to have them separated by tests you will add them to `avocado.test` namespace. 

But we can update our documentation with examples about how to add external libraries logs.